pkg/cvo/availableupdates.go

```diff
@@ -428,34 +428,52 @@ func unknownExposureMessage(risk configv1.ConditionalUpdateRisk, err error) stri
 	return fmt.Sprintf(template, risk.Name, err, risk.Name, risk.Message, risk.Name, risk.URL)
 }
 
+func newRecommendedStatus(now, want metav1.ConditionStatus) metav1.ConditionStatus {
+	switch {
+	case now == metav1.ConditionFalse || want == metav1.ConditionFalse:
+		return metav1.ConditionFalse
+	case now == metav1.ConditionUnknown || want == metav1.ConditionUnknown:
+		return metav1.ConditionUnknown
+	default:
+		return want
+	}
+}
+
+const (
+	recommendedReasonAsExpected       = "AsExpected"
```

Comment:
What do you think of `AvailableUpdate`

Reply:
Or RecommendedUpdate instead of AsExpected

Reply:
`AsExpected` is just an existing string literal extracted to a const, but I guess we can improve it in this PR.

How about `RisksDoNotApply`? Update is recommended *because* risks do not apply?

Reply:
`RisksDoNotApply` sounds better for sure.

Reply:
Perfect, added a commit